pages/docs/how-to-guides/links.mdx

```diff
@@ -56,8 +56,50 @@ To add more links add another object inside the links collection. For example:
 
 _Clicks on these social shortcuts that match any of your links will also increment their respective counters_
 
-### What Links with grouping looks like
+### Grouping your Links (optional)
+To place your social links into categories, you can use names like `primary`,  `secondary`, `business`, or `personal`in the `group` string. Here's an example:
```

Comment:
The names do not have to be like codenames, I think it would be good to add a example over here like the ones mentioned in the example

```suggestion
To place your social links into categories, you can use names like `Business`,  `Socials Primary`, or `:sparkles: Personal` in the `group` string.

These can include any alphanumeric, unicode characters and symbols.
```

+++I think we should also attach a screenshot to tell them how the text they enter inside the `group` field will be displayed

Reply:
Good point @krishstwt! :)